### DISCUSSION TITLE
RICHIESTA URGENTE PER ERRORE SCRUTINIO

### DISCUSSION THREAD

**Discussion by gianlucaaiello56** [iisgiua/giuaschool#515] [Richieste di aiuto] [answered]
Gentilissimi, @bfabio @trinko @zerai @MatteoGheza 
con l'istituto stiamo riscontrando ENORMI problemi con la funzione scrutinio in quanto non possiamo avviarlo in quanto ricevimento "errore 500".
Come potrete immaginare al 6 di giugno per un istituto è molto grave in quanto abbiamo dovuto annullare lo scrutinio online.
Allego il file "log errori" per una maggiore visione.
Attendo risposta.
Saluti.
[2023-06-06_14-00.log](https://github.com/iisgiua/giuaschool/files/11665662/2023-06-06_14-00.log)

**Comment by MatteoGheza** [comment]
Dovete controllare i permessi delle cartelle della cache: `var/cache` e `var/cache/prod`
L'utente del web server deve avere i permessi di scrittura nelle cartelle di cache per funzionare.

**Reply by federicosss** [reply]
E come si fanno a dare i permessi ai docenti coordinatori per eseguire gli scrutini 
Anche io ho questo problema

**Reply by trinko** [reply]
> Dovete controllare i permessi delle cartelle della cache: `var/cache` e `var/cache/prod` L'utente del web server deve avere i permessi di scrittura nelle cartelle di cache per funzionare.

No, l'errore riportato nei log sulla cache non è rilevante. Quel tipo di problema non provoca nessun messaggio di errore visto dall'utente né blocca la procedura.

**Comment by federicosss** [comment]
> Gentilissimi, @bfabio @trinko @zerai @MatteoGheza con l'istituto stiamo riscontrando ENORMI problemi con la funzione scrutinio in quanto non possiamo avviarlo in quanto ricevimento "errore 500". Come potrete immaginare al 6 di giugno per un istituto è molto grave in quanto abbiamo dovuto annullare lo scrutinio online. Allego il file "log errori" per una maggiore visione. Attendo risposta. Saluti. [2023-06-06_14-00.log](https://github.com/iisgiua/giuaschool/files/11665662/2023-06-06_14-00.log)

Ciao per caso sei riuscito a risolvere nel tuo istituto ?

**Comment by trinko** [comment, marked as answer]
Ricordo a tutti che per problemi simili SI DEVE APRIRE UNA SEGNALAZIONE DI ERRORE (ISSUE): [vai alla segnalazione](https://github.com/iisgiua/giuaschool/issues/new?assignees=&labels=Errore&template=bug-report.md&title=)
E si dovrà:
- assicurarsi di avere l'ultima versione rilasciata (attualmente la 1.5.4)
- scrivere nel dettaglio dove si è verificato l'errore (in quale passo della procedura, facendo cosa)

Più si è dettagliati, meglio è.

Aspetto le segnalazioni per analizzare il problema.

**Comment by gianlucaaiello56** [comment]
@trinko quindi come posso risolvere ?

**Reply by trinko** [reply]
> @trinko quindi come posso risolvere ?

Cosa? Hai aperto la segnalazione? Ogni problema deve avere una segnalazione distinta (ISSUE) altrimenti è solo confusione che fa perdere tempo.
Io non ho tempo da perdere, visto che nessuno mi paga per aiutarvi.

**Reply by gianlucaaiello56** [reply]
va bene la apro, il mio problema era il primo "Richiesta urgente per errore scrutinio"
